pom.xml

```diff
@@ -163,11 +163,22 @@
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>scm-api</artifactId>
       <version>${scm-api-plugin.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.jenkins-ci.plugins.workflow</groupId>
+          <artifactId>workflow-step-api</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <artifactId>workflow-step-api</artifactId>
+      <version>2.10</version>
```

Comment:
I thought the workflow 2.x series required upgrading code to 2.7+ (fine if it doesn't... but we should confirm)

Reply:
Nope, workflow 2.x is just the mono repo extraction of all the sub-plugins. All versions that I picked has a max core version of 1.642.3, that's why I bumped core dependency to that.